pkg/local_object_storage/blobstor/blobovniczatree/put_test.go

```diff
@@ -0,0 +1,47 @@
+package blobovniczatree_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/blobovniczatree"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/common"
+	oidtest "github.com/nspcc-dev/neofs-sdk-go/object/id/test"
+	objecttest "github.com/nspcc-dev/neofs-sdk-go/object/test"
+	"github.com/stretchr/testify/require"
+)
+
+func tempDir(tb testing.TB) string {
```

Comment:
how about `t.TempDir()` instead of that func?

Reply:
replaced